KAP/kap-1.md

```diff
@@ -0,0 +1,142 @@
+
+2. **Identifying a Champion**: Every KAP must have a champion—a proponent who takes the responsibility of drafting the proposal, facilitating discussions within the appropriate forums, and striving to build consensus within the community. The champion (also known as the Author) should initially assess whether the idea is suitable for a KAP. Posting the idea to the relevant discussion category or venue within the community, such as the [KAP Discussions](https://github.com/kabuto-sh/proposals/discussions/categories/ideas) or [HBAR Foundry Zulip Chat](https://zulip.kabuto.sh/), is a recommended approach to vetting the idea. This helps save time by ensuring the idea is original and has potential for acceptance within the broader community.
+
+3. **Vetting the Idea Publicly**: The Champion's role includes vetting the idea publicly before developing a full-fledged KAP. This step aims to prevent investing considerable effort in a proposal that might be rejected based on prior discussions or might be applicable only to the proposer. Seeking community feedback ensures the idea's relevance and applicability across various areas where Kabuto technologies are utilized.
+
+4. **Drafting the KAP**: Once the Champion has received initial community feedback indicating the idea's viability for a KAP, they proceed to draft the KAP proposal. This draft is presented in the appropriate venue or platform, ensuring that it adheres to the prescribed KAP format, is of high quality, properly formatted, and adequately addresses initial concerns or considerations raised during the vetting phase.
+
+Starting with a well-vetted and refined idea and proceeding to craft a comprehensive KAP ensures that the proposal is clear, relevant, and aligns with the community's consensus and needs within the Kabuto ecosystem.
+
+### Submitting a KAP
+
+1. **Identification of Co-Authors and Editors**: Though not required, every KAP should have multiple authors or an author and multiple editors. This represents a seed of community involvement in the KAP. These participants can be cultivated from the discussions about the idea around which the KAP is written.
+
+2. **Role of a Sponsor**: The ideal scenario involves identifying a developer as a sponsor. The sponsor serves as a guide and mentor for the KAP author, providing support through the logistical aspects of the KAP process as well as providing an execution path. Being a sponsor does not preclude the individual from becoming a co-author in the future. The name of the sponsor is recorded in the "Sponsor:" field of the KAP header. An example of an ideal sponsor would be a wallet development company sponsoring a KAP written by one of its founders or partners.
```

Comment:
Some slight confusing wording in this paragraph I feel. A sponsor would serve as a 'guide and mentor', which feels counterintuitive with the example of an ideal sponsor (a wallet company sponsoring their own founder). Are there not two roles here? One would be a Sponsor-mentor/backer and the other would be a Sponsor-executor. Maybe there is a top level role 'Sponsor' (eg. Jason underwrites my KAP and also ensures a way for execution), and two subroles Sponsor-backer (any developer(s) who will support and guide the KAP author) and Sponsor-executor (The wallet provider who is going to implement it), whereas the both can be one and the same (where you would use the main role as sponsor).
*Role of a Sponsor**: The ideal scenario involves identifying a developer as a sponsor. The sponsor serves as a guide and mentor for the KAP author, providing support through the logistical aspects of the KAP process as well as providing an execution path. Being a sponsor does not preclude the individual from becoming a co-author in the future. The name of the sponsor is recorded in the "Sponsor:" field of the KAP header. An example of an ideal sponsor would be a wallet development company sponsoring a KAP written by one of its founders or partners.

Reply:
Maybe the sponsor-backers would be co-authors. But then I would remove the reference of 'guide and mentor' from the sponsor description.

Reply:
So...I wanted to capture two unofficial roles: Champion & Sponsors.

Champions are the ones defining the KAP. This is likely one of the authors or co-authors.

Sponsors are the parties that would adopt or benefit from the KAP and provide feedback and "pressure" to the Champion to see it through. This party could also act as an editor, but I think the point is that a sponsor adds credibility to potential execution success.

I feel like I'm trying to put too much in too few paragraphs.

Reply:
yes agree, how about this (need some native english editing ;) ):

2. **Role of a Sponsor**: The ideal scenario involves identifying a end-benefiting party as a sponsor. The sponsor provides support through the logistical aspects of the KAP process as well as providing an execution path. The name of the sponsor is recorded in the "Sponsor:" field of the KAP header. An example of an ideal sponsor would be a wallet development company sponsoring a KAP about a wallet-related proposal.

Reply:
I think that is closer to what I'm trying to say. I'll udpate.